### PR TITLE
feat: Add badge for monthly download stats

### PR DESCRIPTION
**Pull request by KShivendu**
Add a badge for monthly download stats. 29k/month is a decently good number and will attract more developers to try out Qdrant :)

I created a similar PR for Python as well: https://github.com/qdrant/qdrant-client/pull/250

- https://img.shields.io/npm/dm/@qdrant/js-client-rest -> Monthly
- https://img.shields.io/npm/dw/@qdrant/js-client-rest -> Weekly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <a href="https://github.com/qdrant/qdrant-client/blob/master/LICENSE"><img src="https://img.shields.io/badge/License-Apache%202.0-success" alt="Apache 2.0 License"></a>
     <a href="https://qdrant.to/discord"><img src="https://img.shields.io/badge/Discord-Qdrant-5865F2.svg?logo=discord" alt="Discord"></a>
     <a href="https://qdrant.to/roadmap"><img src="https://img.shields.io/badge/Roadmap-2023-bc1439.svg" alt="Roadmap 2023"></a>
+    <a href="https://www.npmjs.com/package/@qdrant/js-client-rest"><img src="https://img.shields.io/npm/dm/@qdrant/js-client-rest" alt="Monthly download stats"></a>
```
Comment (IvanPleshkov) [inline README.md:17]: Hi! Thanks for your proposal! I have a question regarding this badge. We introduced also grpc api. Can we calculate sum of rest and grpc?

Comment (KShivendu) [reply README.md:17]: Unfortunately, not. We'll have to show two badges - which doesn't sound like a good idea. 

Plus, given that gprc downloads are relatively low (371 downloads/month), I think it's okay to leave it for now.

Comment (KShivendu) [reply README.md:17]: Unfortunately, not. We'll have to show two badges - which doesn't sound like a good idea. 

Plus, given that gprc downloads are relatively low (371 downloads/month), I think it's okay to leave it for now.

Comment (KShivendu) [reply README.md:17]: I just deployed a small script on Runkit to fix this https://img.shields.io/endpoint?url=https://runkit.io/kshivendu/npm-download-stats/1.0.0?packages=@qdrant/js-client-grpc,@qdrant/js-client-rest :)

Comment (KShivendu) [reply README.md]: limited

Well looks like runkit frequently rate limits shields.io

**Review by IvanPleshkov — COMMENTED** [review]
Hi! Thanks for your PR and for the additional work on sum calculation. But after discussion, our team decided to decline this badge

**Comment by KShivendu** [comment]
@IvanPleshkov Just to be clear, the rate limit issue only happens for the custom runkit deployment that I did for Qdrant. The original https://img.shields.io/npm/dm/@qdrant/js-client-rest never gets a  timeout. If the reason was something else,  feel free to close this again :)

**Comment by IvanPleshkov** [comment]
It's not a reason. We discussed do we want to have this badge at all. That's why we also closed python badge

**Pull request closed without merging**